types/feedparser/index.d.ts

```diff
@@ -149,6 +149,12 @@ declare namespace FeedParser {
         categories: string[];
     }
 
+    interface Enclosure {
+       length: string;
```

Comment:
According to documentation type and length are optional https://github.com/danmactough/node-feedparser

Reply:
You are right. I updated the typings.